### PR TITLE
4 meds 

### PR DESCRIPTION
**Pull request by SenseCode**
Finished medications chart, request to close issue #4

### CODE REVIEW

File: src/encoded/static/components/patient.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ class Patient extends React.Component {
         ];
 
         const crumbsReleased = (context.status === 'released');
-
+        console.log(context.medications);
```
Comment (cmonnom) [inline src/encoded/static/components/patient.js]: Remove console.log statements

File: src/encoded/static/components/patient.js [hunk 2]
```diff
@@ -96,6 +97,17 @@ class Patient extends React.Component {
...
                     </PanelBody>
                 </Panel> }
+
+                {Object.keys(context.medications).length ?
+                      (<Panel>
+                        <PanelHeading>
+                            <h4>Medications over time</h4>
+                        </PanelHeading>
+                        <PanelBody>
+                            <MedicationChart chartId="medicationChart" data={context.medications} chartTitle="Medications results over time" ></MedicationChart>
```
Comment (cmonnom) [inline src/encoded/static/components/patient.js]: Capitalize "over time" to match other titles

File: src/encoded/static/components/medicationChart.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    this.zingchart.render(this.chart);
+  }
+
+  componentDidMount() {
+    this.zingchart = window.zingchart;
+    this.moment = window.moment;
+    this.filterDataFun();
+    this.transformDataFun();
+    this.drawChart();
+    this.unixToDate();
```
Comment (cmonnom) [inline src/encoded/static/components/medicationChart.js]: unixToDate is called in transformDataFun()
Delete this line

**Review by cmonnom — CHANGES_REQUESTED** [review]
Good work (see minor changes needed in medicationChart.js and patient.js).
Notes (future improvements):
Let's harmonize gantt charts (same colors, Y-axis labels, etc.) in a separate issue.
Use Collapsable panels (separate issue)

**Review by cmonnom — COMMENTED** [review]
I've also noticed some overflow issues when zooming in sometimes.

File: src/encoded/static/components/patient.js (hunk 1 above)
Comment (SenseCode) [reply src/encoded/static/components/patient.js]: Slipped fish...

**Review by cmonnom — APPROVED** [review]

**Review by vsmalladi — APPROVED** [review]

**Pull request merged**